.github/workflows/go.yml

```
@@ -31,7 +31,22 @@ jobs:
         cd go-wrappers-master
         sudo mkdir -p /usr/local/lib/dkls
         sudo cp --recursive includes /usr/local/lib/dkls
-        
+
+    - name: Install frost-zm libraries
+      run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2125:warning:1:11: Brace expansions and globs are literal in assignments. Quote it or use an array

(shellcheck)